examples/timeSeries-hand-gestures/sketch.js

```diff
@@ -0,0 +1,217 @@
+/*
```

Comment:
I tried running this, I might have made a mistake but it changed other files too! not sure how to specify which files only to commit but I had to delete all the changes made on the other files

Reply:
It's probably easiest to run the prettifier, and then selectively determine which changes to include. (In GitHub Desktop, this can be done via the context menu as you are exploring changes. You can accept - or revert changes there.)